tests/fuzz/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+set(libs
+    mbedtls
+)
+
```

Comment:
Not a blocker, but it would be good to support building the fuzzers when we build the library with `make` without using CMake.